src/components/Alerts/BannerAlert.tsx

```diff
@@ -0,0 +1,24 @@
+import React from "react"
+import ReactDOM from "react-dom"
+import { BannerAlertProps } from "./types"
+import { BannerAlertWrapper } from "./StyledAlert"
+
+const BannerAlert = (props: BannerAlertProps) => {
+  const alertMountNode = document.createElement("div")
```

Comment:
Instead of creating a new node in the document, can we instead use the application root node?

This node will otherwise keep lying in the DOM unused, and more will be created everytime we render this

Reply:
Or make it configurable to use a mount node, we can supply mount node form the app